pkg/asset/agent/image/agentimage.go

```diff
@@ -92,7 +103,26 @@ func (a *AgentImage) Generate(dependencies asset.Parents) error {
 	return nil
 }
 
+// Fetching the ignition details from igninfo.json file
+func (a *AgentImage) fetchIgnitionInfo(ignInfoJSONPath string) error {
```

Comment:
It's frustrating to me that there is an [existing implementation of this](https://github.com/openshift/assisted-image-service/blob/main/pkg/isoeditor/stream.go#L76-L80) that we are duplicating.